src/form/death/index.ts

```diff
@@ -15,8 +15,8 @@ import {
   getMaritalStatus,
   registrationEmail,
   registrationPhone,
-  divider,
-  getOccupation
+  divider
+  // getOccupation
```

Comment:
```suggestion
```

Reply:
I don't know if it's there just as a guideline for the implementer perhaps?